deploy/deploy.js

```diff
@@ -21,6 +22,12 @@ module.exports = async ({ deployments, getNamedAccounts }) => {
         from: deployer,
         contract: contractName,
         skipIfAlreadyDeployed: true,
```

Comment:
question: @ZumZoom what is it good practice to use "args" or it's better to change how deploy is invoked